engine/table/src/main/java/io/deephaven/engine/table/impl/sources/regioned/RegionedColumnSourceManager.java

```diff
@@ -318,13 +363,15 @@ private WritableRowSet update(final boolean initializing) {
             }
         }
 
-        final int previousNumRegions = includedTableLocations.size();
-        final int newNumRegions = previousNumRegions + (entriesToInclude == null ? 0 : entriesToInclude.size());
+        final RowSetBuilderSequential addedRegionBuilder = RowSetFactory.builderSequential();
```

Comment:
Let's not allocate builders we won't use. If `entriesToInclude.isEmpty()`, use `null`. Similar for removed.